tests/src/test/scala/common/WskCliOperations.scala

```diff
@@ -62,18 +65,17 @@ import whisk.utils.retry
  * It also sets the apihost and apiversion explicitly to avoid ambiguity with
  * a local property file if it exists.
  */
-class Wsk() extends RunWskCmd with BaseWsk {
-  override implicit val action = new WskAction
-  override implicit val trigger = new WskTrigger
-  override implicit val rule = new WskRule
-  override implicit val activation = new WskActivation
-  override implicit val pkg = new WskPackage
-  override implicit val namespace = new WskNamespace
-  override implicit val api = new WskApi
+class Wsk extends WskOperations {
```

Comment:
I adopted the convention of `<Entity>Operations` for the base type, and prefixed `Cli` or `Rest` for the respective variants.